microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -496,6 +497,23 @@ def _configure_vdpa_devices(self, node: Node) -> None:
             node.tools[Chown].change_owner(file=PurePath(device_path), user=user)
             node.tools[Chmod].chmod(path=device_path, permission=permission, sudo=True)
 
+    def _set_data_disk(self) -> None:
+        datadisk_name = ""
+        disks = self.node.tools[Lsblk].get_disks()
+        # get the first unmounted disk (data disk)
+        for disk in disks:
+            if disk.is_mounted:
+                continue
+            if disk.name.startswith("sd"):
+                datadisk_name = disk.device_name
+                break
+        # running lsblk once again, just for human readable logs
+        self.node.execute("lsblk", shell=True)
+        if not datadisk_name:
+            raise LisaException("No data disk found")
```

Comment:
Specify which name is looking from lsblk

Reply:
We are not looking for specially device just starting with `sd`. Instead, we are looking for a data disk which is unmounted.
I feel this exception message is good enough for debugging the failures.

Reply:
The error message is for people, who run tests but never read the code. So, it needs to include as much information as possible to reduce their troubleshooting cost.

Reply:
Updated the message to `No unmounted data disk (/dev/sdX) found`